target/linux/qualcommax/ipq807x/base-files/etc/board.d/01_leds

```diff
@@ -9,6 +9,7 @@ case "$board" in
 arcadyan,aw1000)
 	ucidef_set_led_netdev "5g" "5G" "green:5g" "wwan0"
 	ucidef_set_led_netdev "wan" "WAN" "green:internet" "wan"
```

Comment:
What is this LED as I dont see it being registered in the DTS?

Reply:
This is the LED on the panel, see: https://github.com/openwrt/openwrt/blob/main/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8072-aw1000.dts#L124